src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
+                tasks[taskNumber - 1].setDone(false);
+                printUnmarkedAcknowledgement(tasks, taskNumber);
+                break;
+
+            case "todo" :
+                System.out.println("    _________________________________________");
+                System.out.println("    Please specify task: ");
+                inputString = getInputString();
+                tasks[counter] = new Todo(inputString);
+                tasks[counter].print();
+                counter++;
+                System.out.println("    Now you have " + counter + " tasks in your list!");
+                break;
+
+            case "deadline" :
+                System.out.println("    _________________________________________");
+                System.out.println("    Please specify task: ");
+                inputString = getInputString();
+                inputString += '\0';
+                int deadlinePosition = inputString.indexOf("/");
+                int endPosition = inputString.indexOf("\0");
+                String taskName = inputString.substring(0, deadlinePosition);
+                String deadline = inputString.substring(deadlinePosition + 1, endPosition);
+                tasks[counter] = new Deadline(taskName, deadline);
+                tasks[counter].print();
+                counter++;
+                break;
+
+            case "event" :
+                System.out.println("    _________________________________________");
+                System.out.println("    Please specify task: ");
+                inputString = getInputString();
+                inputString += '\0';
+                int deadlineStartPosition = inputString.indexOf("/");
+                int deadlineEndPosition = inputString.indexOf("|");
+                endPosition = inputString.indexOf("\0");
+                taskName = inputString.substring(0, deadlineStartPosition);
+                String deadlineStart = inputString.substring(deadlineStartPosition + 1, deadlineEndPosition);
+                String deadlineEnd = inputString.substring(deadlineEndPosition + 1, endPosition);
+                tasks[counter] = new Event(taskName, deadlineStart, deadlineEnd);
+                tasks[counter].print();
+                counter++;
+                break;
+
+            default:
+                in = new Scanner(System.in);
+                inputString = in.nextLine();
+                break;
+            }
+        }
+
+        //When user types "bye"
+        System.out.println("    Bye. Hope to see you again soon!");
+        System.out.println("    _________________________________________");
+        System.out.println("     ");
+    }
+
+    private static String getInputString() {
+        Scanner in;
+        String inputString;
+        in = new Scanner(System.in);
+        inputString = in.nextLine();
+        return inputString;
+    }
+
+    private static void printUnmarkedAcknowledgement(Todo[] tasks, int taskNumber) {
```

Comment:
I like the functions here to help you to print the messages. The name format is also good.